Gemfile

```diff
@@ -2,9 +2,9 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in colonel.gemspec
 gemspec
-ruby '2.1.3'
+ruby '2.2.8'
```

Comment:
Can we update ruby to 2.3.5 as 2.2.8 `will EOL soon!` https://www.ruby-lang.org/en/downloads/